pkg/storaged/format-dialog.jsx

```diff
@@ -165,6 +161,26 @@ export function format_dialog(client, path, start, size, enable_dos_extended) {
     if (create_partition && enable_dos_extended)
         add_fsys(true, { value: "dos-extended", title: _("Extended partition") });
 
+    function is_encrypted(vals) {
+        return vals.crypto !== "none";
+    }
+
+    function add_crypto_type(value, title) {
+        if ((client.manager.SupportedEncryptionTypes && client.manager.SupportedEncryptionTypes.indexOf(value) != -1) ||
+            value == "luks1") {
+            crypto_types.push({
+                value: value,
+                title: title + (value == client.manager.DefaultEncryptionType
+                    ? " - " + _("Recommended default")
```

Comment:
I would add it as a helperText bellow the select, but this can gladly be a followup or not at all.

Reply:
This follows the same style we have for the filesystem type.